### DISCUSSION TITLE
Certificato delle competenze

### DISCUSSION THREAD

**Discussion by sempliciniluigi89** [iisgiua/giuaschool#442] [Idee e proposte]
Sul certificato delle competenze viene riportato un N°.................,allora senza che la segreteria vada a ricordare numero per numero ,non sarebbe possibile generare il numero del certificato in automatico?
Inoltre ,sullo stesso certificato è presente "nell'assolvimento dell'obbligo di istruzione ,della durata di 10 anni" , ora io mi chiedo perchè 10 e non 5 dato che 5 sono gli anni delle superiori?

**Comment by manuelbarberi** [comment]
Perchè considera anche le elementari

**Comment by trinko** [comment]
Si, per il numero, in effetti si potrebbe fare. Lo segno come richiesta.
